docs/tutorials/03-model-serving.ipynb

```diff
@@ -352,7 +376,7 @@
     "<a id=\"deploy-serving\"></a>\n",
     "## Deploy the serving function\n",
     "\n",
-    "Deploy the serving function and use `invoke` to test it with the provided `sample`."
+    "Deploy the serving function and use `invoke` to test it with the provided `sample` by 2 API's `infer` & `infer_dict`."
```

Comment:
"Deploy the serving function and use `invoke` to test it with the provided `sample` by using 2 API's `infer` & `infer_dict`."